plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/compare/changedList/ChangedFileNode.java

```diff
@@ -27,24 +28,33 @@
  *
  * @author Igor Vinokur
  */
-public class ChangedNode extends AbstractTreeNode implements HasPresentation, HasAction {
-
-    private String name;
-    private String state;
+public class ChangedFileNode extends AbstractTreeNode implements HasPresentation, HasAction {
```

Comment:
You implement `HasAction` interface, but method body is empty, do you really need to implement this feature?